src/tools/miri/src/machine.rs

```diff
@@ -1241,4 +1261,23 @@ impl<'mir, 'tcx> Machine<'mir, 'tcx> for MiriMachine<'mir, 'tcx> {
         }
         res
     }
+
+    fn dump_backtrace(
+        ecx: &InterpCx<'mir, 'tcx, Self>,
+        id: AllocId,
+        alloc: &Allocation<Self::Provenance, Self::AllocExtra, Self::Bytes>,
+    ) -> InterpResult<'tcx, ()> {
```

Comment:
do you expect this to ever be able to fail?

Reply:
The only caller `unwrap`s anyway. Seems better to use a `()` return type here.